.ado/continuous.yml

```diff
@@ -13,13 +13,13 @@ parameters:
     default:
       Small:
         name: rnw-pool-2
-        demands: ImageOverride -equals rnw-img-node16
+        demands: ImageOverride -equals rnw-img-test
       Medium:
         name: rnw-pool-4
-        demands: ImageOverride -equals rnw-img-node16
+        demands: ImageOverride -equals rnw-img-test
       Large:
         name: rnw-pool-8
-        demands: ImageOverride -equals rnw-img-node16
+        demands: ImageOverride -equals rnw-img-test
```

Comment:
Will revert before merge. Wanted to test impact of installing 7zip on the machine, and that change is now being applied to the main image.